src/couch_replicator_scheduler.erl

```diff
@@ -205,52 +213,62 @@ handle_config_terminate(Self, _, _) ->
 
 %% private functions
 
-start_jobs(Count) ->
+start_jobs(Count, MaxHistory) ->
     Runnable0 = pending_jobs(),
     Runnable1 = lists:sort(fun oldest_job_first/2, Runnable0),
     Runnable2 = lists:filter(fun not_recently_crashed/1, Runnable1),
     Runnable3 = lists:sublist(Runnable2, Count),
-    lists:foreach(fun start_job_int/1, Runnable3).
+    lists:foreach(fun(J) -> start_job_int(J, MaxHistory) end, Runnable3).
```

Comment:
A list comprehension could do it then. Just return ok, in the end to stay consistent with the existing return type